website/docs/releases/2024/v2024.8.md

```diff
@@ -9,6 +9,13 @@ slug: "/releases/2024.8"
 To try out the release candidate, replace your Docker image tag with the latest release candidate number, such as 2024.8.0-rc1. You can find the latest one in [the latest releases on GitHub](https://github.com/goauthentik/authentik/releases). If you don't find any, it means we haven't released one yet.
 :::::
 
+## Highlights
+
+-   **Source property mappings for SCIM, OAuth, SAML and Plex sources**: All sources now support property mappings to configure how authentik interprets the data, and groups can be synced from all sources that provide group info.
+-   **RADIUS provider custom attribute support**: Integrate RADIUS better into network infrastructure by sending vendor-specific attributes based on the user authenticating.
+-   **SAML encryption support**: SAML source and provider support encryption now, which prevents anyone from viewing the information of Assertions in-flight.
```

Comment:
```suggestion
-   **SAML encryption support**: SAML source and provider data is now encrypted, which prevents anyone from viewing the information of Assertions in-flight.
```

Reply:
This is an optional feature so saying "data is now encrypted" is not true